packages/web-service/src/server.js

```diff
@@ -134,6 +135,11 @@ const init = async server => {
   // Register the dynamic routes
   await server.route(routes)
 
+  // If the directory doesn't exist to hold our files we need to scan, create it
```

Comment:
SCANDIR  must be an absolute path for it to work in the live environment. Locally you can give it anything - perhaps /tmp